nav2_bringup/bringup/launch/multi_tb3_simulation_launch.py

```diff
@@ -110,7 +110,7 @@ def generate_launch_description():
 
     # Start Gazebo with plugin providing the robot spawing service
     start_gazebo_cmd = ExecuteProcess(
-        cmd=[simulator, '--verbose', '-s', 'libgazebo_ros_factory.so', world],
+        cmd=[simulator, '--verbose', '-s', 'libgazebo_ros_init.so', '-s', 'libgazebo_ros_factory.so', world],
```

Comment:
Can you make this adjustment in all of the launch files so that we don't run into this issue again when someone tries to modify it to their needs?